frappe/public/build.json

```diff
@@ -216,7 +219,12 @@
 		"public/js/frappe/list/listview.js",
 		"public/js/frappe/views/calendar_base.js",
 		"public/js/frappe/views/calendar.js",
-		"public/js/frappe/views/ganttview.js"
+		"public/js/frappe/views/ganttview.js",
+
+		"public/js/frappe/list/imageview.js",
```

Comment:
load this files on demand with `frappe.require` this is too much load incase there are no image views!